static/css/main.css

```diff
@@ -151,6 +151,11 @@ body {
  }
 
  /*Shared Experience Section*/
+
+ .alert-container {
+  padding-top: 10px;
+ } 
```

Comment:
The alerts are slightly less wide than the containers directly below, maybe it would be possible to have them the same width?

Reply:
ah yes, that makes sense!